reference/functions/hostswithclass.markdown

```diff
@@ -18,7 +18,7 @@ hostnames, ip-addresses or public key SHAs of hosts that have a given class.
 configuration files for (classes of) hosts. Hosts are selected based on the
 classes set during the most recently collected agent run.
 
-[%CFEngine_function_attributes(class, field, hostkey)%]
+[%CFEngine_function_attributes(class, field)%]
```

Comment:
where is the list of options for field?

Reply:
As Nick says, the list of options is what the macro is for, essentially. It gets it from the syntax description (C code).

The change here is correct, the hostswithclass function has 2 arguments, not 3.